awx/ui/src/screens/Credential/Credential.js

```diff
@@ -22,6 +22,16 @@ import { CredentialsAPI } from 'api';
 import CredentialDetail from './CredentialDetail';
 import CredentialEdit from './CredentialEdit';
 
+const jobTemplateCredentialTypes = [
+  'machine',
+  'cloud',
+  'net',
+  'ssh',
+  'vault',
+  'kubernetes',
+  'cryptography',
+];
```

Comment:
Curious, how is this list determined?

Reply:
Job templates only support certain types of credentials and the api doesn't have a way of telling the UI, which types are JTs support so the UI has the be the source of truth for that information.